src/lib/voice-gateway/credential-file.ts

```
@@ -31,39 +30,75 @@ function validateBearer(value: string, label: string): void {
+        descriptors.deployment,
+        "Voice gateway deployment credential",
+      ),
+      openClawCredential: readBearer(descriptors.openClaw, "Voice gateway OpenClaw credential"),
+    };
   } finally {
-    if (descriptor !== undefined) fs.closeSync(descriptor);
+    for (const descriptor of uniqueDescriptors) {
+      try {
+        fs.closeSync(descriptor);
+      } catch (error) {
+        if ((error as NodeJS.ErrnoException).code !== "EBADF") throw error;
```

Biome (2.5.6)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)